src/core/src/main/java/org/apache/jmeter/JMeterGuiLauncher.kt

```diff
@@ -83,6 +83,7 @@ public object JMeterGuiLauncher {
     }
 
     private suspend fun startGuiInternal(testFile: String?) {
+        JMeterUtils.applyHiDPIOnFonts()
```

Comment:
There's the same `applyHiDPIOnFonts` call a couple of lines below (right after `setProgress(1)`). Could you please clarify why adding one more improves things?